### ISSUE TITLE
Streamlit support is broken for `streamlit>0.78`

### ISSUE THREAD

**Issue by pietrolesci** [rstudio/rsconnect-python#187]
Hi there,

I am trying to deploy a streamlit app on the rstudio-connect server. The app was running fine with `streamlit==0.78` but fails with any later version of that library with the following error visible at the url where the app should be deployed:

```bash
The application failed to start: exited unexpectedly with code 1

2021/07/08 16:12:11.440510803 Unexpected error while running Streamlit app: run() missing 1 required positional argument: 'flag_options'
2021/07/08 16:12:11.440561807 Traceback (most recent call last):
2021/07/08 16:12:11.440708568   File "/opt/rstudio-connect/python/connect_streamlit_runtime.py", line 119, in <module>
2021/07/08 16:12:11.440720863     bootstrap.run(filename, "", args)
2021/07/08 16:12:11.440753871 TypeError: run() missing 1 required positional argument: 'flag_options'
```

**Comment by mmarchetti** [comment]
This is a known issue due to a change in Streamlit and is fixed in RStudio Connect 1.8.8. Workaround for earlier Connect versions is to pin streamlit to version 0.78.

**Comment by pietrolesci** [comment]
@mmarchetti thanks a lot for your reply :)
